.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
see my updated version, which I think is a compromise between the two you suggested.

I'm not sure I see much value derived from adding the environment variables, since we're only dealing with 2 versions.

It could simplify the two ansible environments to just using `{[testenv]deps}`, I suppose...